DESCRIPTION

```diff
@@ -22,8 +22,11 @@ Imports:
     dplyr,
     ggplot2,
     glue,
+    grid,
+    gridExtra,
     magrittr,
     patchwork,
+    png,
```

Comment:
just a note: I love that we can rely on these new packages being automatically picked up in the Docker image when this pkg is installed, rather than having to manually add them. 😁 @jdhoffa

Reply:
Do you mean because they are in the description?

Reply:
yes! and because we are installing it as a package (rather than just copying in the repo and accessing the R files/functions directly)

Reply:
packages are great sometimes ;)